pkg/controller/perconaservermongodbbackup/perconaservermongodbbackup_controller.go

```diff
@@ -195,7 +197,8 @@ func (r *ReconcilePerconaServerMongoDBBackup) checkFinalizers(cr *api.PerconaSer
 					continue
 				}
 
-				err = b.pbm.C.DeleteBackup(cr.Status.PBMname)
+				e := b.pbm.C.Logger().NewEvent(string(pbm.CmdDeleteBackup), "", "", primitive.Timestamp{})
```

Comment:
Why do we need to pass empty Timestamp and why do we even need to create the `Event`. Can we just pass `nil`?

Reply:
I assumed this is the most similar behavior as before the update.

In the PBM repo, an epoch is either empty or from config. We can get it from config via `PBM.GetEpoch()` function.

Do you think it would be better?

Reply:
@dAdAbird said your approach is ok. So I'll just resolve it.